ci-operator/step-registry/ipi/conf/aws/usage-info/ipi-conf-aws-usage-info-commands.sh

```diff
@@ -44,6 +35,22 @@ usage-pull-request ${pull_number}
 usage-ci-type ${ci_type}
 EOF
 
+# for hypershift mgmt cluster, store one more cluster info file for the hosted cluster
+if [[ "${USAGE_CLUSTER_TYPE}" == "hypershift-mgmt" ]]; then
+  hosted_cluster_file="${SHARED_DIR}/hosted_cluster_info_file"
+  cp "${cluster_info_file}" "${hosted_cluster_file}"
+  sed -ie 's|usage-cluster-type.*|usage-cluster-type hypershift-hosted|' "${hosted_cluster_file}"
+  exit 0
```

Comment:
When running `hypershift-mgmt` cluster install, once come to this line, the whole step would exit without any user tag are patched for mgmt cluster vms.

Reply:
ah, yes.. I need update this part and check the mgmt tags too ..

Reply:
do you think it is ok by just removing` exit 0`

Reply:
I guess so